### PR TITLE
Autoarima details

### PR DESCRIPTION
**Pull request by sdtaylor**

### CODE REVIEW

**Comment by davharris** [comment]
LGTM. Thanks @sdtaylor!

File: manuscript/draft.Rmd [hunk 1]
```diff
@@ -307,7 +307,7 @@ including trends). The `auto.arima` function chooses whether to include each of
 these components (and how many terms to include for each one) using AICc
 [@forecast1]. Since there is no seasonal component to the BBS time-series, we
 did not include a season component in these models. Otherwise we used the
-default settings for this function [@forecast1].
+default settings for this function (See suppliment for details).
```
Comment (ethanwhite) [inline manuscript/draft.Rmd]: suppliment -> supplement

File: manuscript/reviewer_reply.Rmd [hunk 2]
```diff
@@ -76,6 +76,8 @@
 
 *Line 183: Because these defaults may change over time, it’d be good to specify things like the ranges of AR / MA order that you searched over, which criteria was used to compare models, etc. And whether or not you included a drift parameter?*
 
+>We agree that software can evolve over time and make methods non-repricable. We have add details of the auto.arima model in the suppliment. 
+
```
Comment (ethanwhite) [inline manuscript/reviewer_reply.Rmd]: "have add" -> "added"

**Comment by sdtaylor** [comment]
Thanks, typos fixed.

**Comment by davharris** [comment]
Thanks Shawn!

**Pull request merged**